lib/components/Modal.js

```diff
@@ -5,14 +5,14 @@ var ModalPortal = React.createFactory(require('./ModalPortal'));
 var ariaAppHider = require('../helpers/ariaAppHider');
 var elementClass = require('element-class');
 var renderSubtreeIntoContainer = require("react-dom").unstable_renderSubtreeIntoContainer;
-var Assign = require('lodash.assign');
-
+var Assign = require('../helpers/assign');
 var SafeHTMLElement = ExecutionEnvironment.canUseDOM ? window.HTMLElement : {};
 var AppElement = ExecutionEnvironment.canUseDOM ? document.body : {appendChild: function() {}};
 
 var Modal = React.createClass({
 
   displayName: 'Modal',
+  openBodyClass: "ReactModal__Body--open",
```

Comment:
Your PR #239 will make all classes customizable, so this refactor will be discarded. Better removed those diffs that use this property `openBodyClass`.